notes/lecture5.tex

```diff
@@ -0,0 +1,141 @@
+\section{Lecture 5: Conditional Gradient (Frank-Wolfe)}
+\sectionlabel{frankwolfe}
+In this lecture we discussed about the conditional gradient method, also known as the Frank-Wolfe (FW) algorithm. The motivation of using this approach is the projected gradient descent can be computationally inefficient under certain scenarios.
+
+\subsection{Intuition behind Frank-Wolfe algorithm}
+We motivate this lecture by the computational inefficiency that projected gradient can have. With conditional gradient, we are able to sidestep some of these inefficiencies. An intuitive idea of the FW algorithm is as follows. \\
+\\We start from $x_0$. Then, for t = 1 to T steps, we set
```

Comment:
Math environments missing around `t=1` and `T`, also please don't use `\\` for new lines.